assets/css/style.css

```diff
@@ -186,45 +182,42 @@ h3:hover {
   font-weight: var(--fw-regular);
 }
 
-#projectGrid > div {
-  background: var(--clr-grey);
-}
-
```

Comment:
Didn't seem to be needed since the background is also set for `#projectGrid > *` earlier in the file.